framework/src/bundle/BundleUtils.h

```diff
@@ -20,8 +20,8 @@
 
 =============================================================================*/
 
-#ifndef CPPMICROSERVICES_BUNDLEUTILS_H
-#define CPPMICROSERVICES_BUNDLEUTILS_H
```

Comment:
revert the #define name change. the convention used is CPPMICROSERVICES_<HEADER FILENAME>_H